### PR TITLE
gtp_common: add to tilegrid

### PR DESCRIPTION
**Pull request by acomodi**
Signed-off-by: Alessandro Comodi <acomodi@antmicro.com>

For the time being, the GTP_COMMON is restricted only to the artix7 devices. We can enable them for the other families in a second moment.

### CODE REVIEW

**Comment by acomodi** [comment]
@litghost  The error should be solved with the latest update.

Basically, the 200T device has `GTP_COMMON_MID_[LEFT|RIGHT]` tiles which have bits in frames 0 and 1, instead of 28 and 29 as for the `GTP_COMMON`.

To have the tilegrid fuzzer as generic as possible, I have added an `auto-alignment` for the baseaddress, so that we do not need special if-else to determine whether we are on the 200T or 50/100T.

**Comment by acomodi** [comment]
> Use the AUTO_ALIGN style from prjuray

@litghost Done

File: fuzzers/005-tilegrid/fuzzaddr/common.mk [hunk 1]
```diff
@@ -14,7 +14,7 @@ $(SPECIMENS_OK):
 	touch $@
 
 clean:
```
Comment (litghost) [inline fuzzers/005-tilegrid/fuzzaddr/common.mk:16]: We need a "clean_all" with this change.

Comment (acomodi) [reply fuzzers/005-tilegrid/fuzzaddr/common.mk:16]: Done

**Pull request merged**